app/views/devise/sessions/new.html.erb

```diff
@@ -1,16 +1,18 @@
-<h2>Log in</h2>
 
+
+<div class="govuk-grid-row">
+<div class='govuk-grid-column-two-thirds govuk-!-margin-top-3 govuk-!-margin-bottom-3'>
+<h1 class="govuk-heading-l">Sign in</h2>
 <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  <div class="field govuk-form-group">
+    <%= f.govuk_email_field :email, label: {text: 'Email'}, width: 'one-half' %>
   </div>
 
-  <div class="field">
-    <%= f.label :password %><br />
-    <%= f.password_field :password, autocomplete: "current-password" %>
+  <div class="field govuk-form-group">
```

Comment:
I don't think we need to manually markup this `govuk-form-group` styled container. From the docs at https://govuk-form-builder.netlify.app/form-elements/text-input/#output-html-generating-a-text-input-with-a-custom-label-and-hint it looks like we get the following HTML from the component:

```
<%= f.govuk_text_field :first_name, label: { text: 'First name' }, hint: { text: 'You can find it on your birth certificate' } %>
```

renders

```html
<div class="govuk-form-group">
  <label for="person-first-name-field" class="govuk-label">
    First name
  </label>
  <div class="govuk-hint" id="person-first-name-hint">
    You can find it on your birth certificate
  </div>
  <input id="person-first-name-field" class="govuk-input" aria-describedby="person-first-name-hint" type="text" name="person[first_name]" />
</div>
```